### DISCUSSION TITLE
Postman collection for DB Middleware APIs

### DISCUSSION THREAD

**Discussion by rishi1596** [airavata-courses/garuda#60] [General]
Please share the postman collection for DB Middleware APIs

**Comment by pranavacharya** [comment]
> Please share the postman collection for DB Middleware APIs
[garuda_db_middleware.postman_collection.zip](https://github.com/airavata-courses/garuda/files/8009653/garuda_db_middleware.postman_collection.zip)


This collection only has 2 endpoint. If you add rest, post it here
